Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -742,15 +742,19 @@ protected function getRealpathCacheSize()
      * root directory. To make this command work for every case, read Composer's
      * vendor/ directory location directly from composer.json file.
      *
-     * @return string
+     * @return string Absolute path to vendor directory
      */
     private function getComposerVendorDir()
     {
+        $composerVendorDir = "vendor";
         $composerJson = json_decode(file_get_contents(__DIR__.'/../composer.json'));
```

Comment:
I would decode the JSON as an array to avoid the ugly `$composerJson->config->{'vendor-dir'}` construct below.